rest-api-spec/src/main/resources/rest-api-spec/api/_common.json

```diff
@@ -23,6 +23,10 @@
       "type": "string",
       "description": "The URL-encoded request definition. Useful for libraries that do not accept a request body for non-POST requests."
     },
+    "source_content_type": {
+      "type": "string",
+      "description": "The URL-encoded media type that indicates the format of the URL-encoded request definition passed with the source parameter. For example, application/json, encoded as application%2fjson. Useful for libraries that do not accept a request body for non-POST requests."
```

Comment:
> Useful for libraries that do not accept a request body for non-POST requests.

Can we remove this mention? As a rule any GET API that has a body also has a POST variant.

Reply:
I'm OK with removing it. Probably want to remove the same statement from the `"source"` parameter above too?